wordle-web/src/scripts/wordsService.ts

```diff
@@ -23,6 +25,13 @@ export abstract class WordsService {
     return validWordList
   }
 
+  static async getApiWord(): Promise<string> {
```

Comment:
I don't think this is ever called (your restart game just uses getRandomWord).  I don't think this is an issue for the assignment as the assignment only specifies that it wants the API to deal with the leaderboard so you should be fine to completely ignore this.